<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="digits">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>400</width>
      <height>500</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>87</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>88</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>88</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>300</y>
       <width>87</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>300</y>
       <width>87</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>88</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>160</y>
       <width>87</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>9</string>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>230</y>
       <width>88</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>87</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>87</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="push_C">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>88</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_LS">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>43</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>(</string>
     </property>
    </widget>
    <widget class="QPushButton" name="push_module">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>87</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QPushButton" name="push_divi">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>87</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="push_X">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>87</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QPushButton" name="push_minus">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>230</y>
       <width>87</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="push_plus">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>300</y>
       <width>87</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_eq">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>380</y>
       <width>87</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QPushButton" name="push_neg">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>380</y>
       <width>87</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>+/-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="push_dot">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>380</y>
       <width>87</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_RS">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>43</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>)</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="result_show">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>90</y>
      <width>382</width>
      <height>39</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	qproperty-alignment: 'AlignVCenter | AlignRight ';
	border-bottom: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="history_show">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>6</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	qproperty-alignment: 'AlignVCenter | AlignRight ';
	border-bottom: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>400</width>
      <height>1</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/black.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="skob_podsk">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>110</y>
      <width>10</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: gray;
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <zorder>result_show</zorder>
   <zorder>digits</zorder>
   <zorder>history_show</zorder>
   <zorder>label</zorder>
   <zorder>skob_podsk</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
